static_files/tutorials/javascript/script/svg.cwct/eng/step4/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  'rules': {
+    'no-unused-vars': [2, {
+      'varsIgnorePattern':'^points$'
```

Comment:
You don't need to put an .eslintrc file in each directory, you could just disable the no-unused-vars or defining an general prefix like the general "opt_*" or "*_unused".

Reply:
Do you prefer to disable no-unused-vars for all of the example code?

I'm glad to just bump .eslintrc up a level and disable no-unused-vars. This is example code that will be displayed to the user. The per-step .eslintrc files disables linting for as little as possible in order to enforce standards within the example code.

Reply:
I think its easier to disable "no-unused-vars" for all example code.
But If you want to create an .eslintrc file for each example that's also fine with me.